.github/workflows/tests.yml

```diff
@@ -159,32 +163,45 @@ jobs:
       with:
         distribution: 'temurin'
         java-version: ${{ matrix.java-version }}
-    - name: Set up Gradle 7.3 # must be done manually due to no gradle build in capa
+    - name: Set up Gradle ${{ matrix.gradle-version }} # must be done manually due to no gradle build in capa
       run: |
         mkdir /opt/gradle
-        wget "https://services.gradle.org/distributions/gradle-7.3-bin.zip" -O /opt/gradle/gradle-7.3.zip
-        unzip /opt/gradle/gradle-7.3.zip -d /opt/gradle
-    - name: Install Ghidra 10.3
+        wget "https://services.gradle.org/distributions/gradle-${{ matrix.gradle-version }}-bin.zip" -O /opt/gradle/gradle-7.3.zip
+        unzip /opt/gradle/gradle-${{ matrix.gradle-version }}.zip -d /opt/gradle
+    - name: Install Ghidra ${{ matrix.ghidra-version }} 
       run: |
         mkdir ./.github/ghidra
-        wget "https://github.com/NationalSecurityAgency/ghidra/releases/download/Ghidra_10.3_build/ghidra_10.3_PUBLIC_20230510.zip" -O ./.github/ghidra/ghidra_10.3_PUBLIC.zip
-        unzip .github/ghidra/ghidra_10.3_PUBLIC.zip -d .github/ghidra/
-    - name: Install Jep 4.1.1
+        mkdir ./.github/ghidra/project
+        wget "https://github.com/NationalSecurityAgency/ghidra/releases/download/Ghidra_${{ matrix.ghidra-version }}_build/ghidra_${{ matrix.ghidra-version }}_PUBLIC_20230510.zip" -O ./.github/ghidra/ghidra_${{ matrix.ghidra-version }}_PUBLIC.zip
+        unzip .github/ghidra/ghidra_${{ matrix.ghidra-version }}_PUBLIC.zip -d .github/ghidra/
+        .github/ghidra/ghidra_${{ matrix.ghidra-version }}_PUBLIC/ghidraRun 
+    - name: Install Jep ${{ matrix.jep-version }} 
       run : |
         mkdir ./.github/jep
-        wget "https://github.com/ninia/jep/archive/refs/tags/v4.1.1.zip" -O ./.github/jep/jep-4.1.1.zip
-        unzip .github/jep/jep-4.1.1.zip -d .github/jep/
-        pip install .github/jep/jep-4.1.1/
+        wget "https://github.com/ninia/jep/archive/refs/tags/v${{ matrix.jep-version }}.zip" -O ./.github/jep/jep-${{ matrix.jep-version }}.zip
+        unzip .github/jep/jep-${{ matrix.jep-version }}.zip -d .github/jep/
+        pip install .github/jep/jep-${{ matrix.jep-version }}/
```

Comment:
Thoughts on using `pip install jep==${{ matrix.jep-version }}` to clean this up a bit?

Reply:
Included the pip install. CI is working as expected.